winchan.js

```diff
@@ -197,7 +197,9 @@ var WinChan = (function() {
                 cb = null;
               }
             }
-          } catch(err) { }
+          } catch(err) {
+            console.err(err);
```

Comment:
shouldn't we probably call the callback with it?

Also, I'd probably move the try/catch only for the `JSON.parse()` so we don't catch here errors behind the matrix.